docs/source/leapp-tool.rst

```diff
@@ -53,23 +53,25 @@ migrate-machine
 destroy-containers
```

Comment:
Hmm, we should probably delete the `list-machines` docs above, and add `check-target` docs.